bin/eslint.js

```diff
@@ -124,7 +124,18 @@ ${message}`);
 
     // Call the config initializer if `--init` is present.
     if (process.argv.includes("--init")) {
-        await require("../lib/init/config-initializer").initializeConfig();
+
+        // `eslint --init` has been moved to `@eslint/config`
+        console.warn("You can also run this command directly using 'npm init @eslint/config'.");
+
+        const spawn = require("cross-spawn");
+        const npmProcess = spawn.sync("npm", ["init", "@esllint/config"], { encoding: "utf8" });
+
+        if (npmProcess.status === 0) {
+            console.log(npmProcess.stdout);
+        } else {
+            console.error(npmProcess.stderr);
+        }
```

Comment:
Since the init process prompts the user interactively, we'll need to inherit stdio instead.

```suggestion

        const npmProcess = spawn.sync("npm", ["init", "@eslint/config"], { encoding: "utf8", stdio: "inherit" });
```